frameworks/elastic/src/main/dist/elasticsearch.yml

```diff
@@ -19,6 +19,11 @@ bootstrap.memory_lock: false
 metrics.statsd.host: {{STATSD_UDP_HOST}}
 metrics.statsd.port: {{STATSD_UDP_PORT}}
 
+{{#DETECT_ZONES}}
+node.attr.zone: {{ZONE}}
+cluster.routing.allocation.awareness.attributes: zone
```

Comment:
Full namespacing to node.attr.zone is not required?

Reply:
[No](https://www.elastic.co/guide/en/elasticsearch/reference/current/allocation-awareness.html).